### ISSUE TITLE
Potential new feature | ability to configure generate file name (naming convention)

### ISSUE THREAD

**Issue by mlukasik-branch** [egm0121/vs-jest-test-gen#1]
Generated file names are suffixed with "generated.test.ts".

Is it hardcoded?
Asking cause in my case it would help a lot to have option to configure it.

Example:
1. remove "generated" word
2. add "pact.test.js" suffix to distinguish API contract tests.

**Comment by egm0121** [comment]
I agree sounds like a good addition to have settings page for the extension where you can configure:
--fileSuffix
and 
--outputDir
as supported by the underlying CLI module [jest-test-gen](https://github.com/egm0121/jest-test-gen/blob/master/CHANGELOG.md)
